extensions/MESA/MESA_framebuffer_flip_x.txt

```diff
@@ -0,0 +1,91 @@
+Name
+
+    MESA_framebuffer_flip_x
+
+Name Strings
+
+    GL_MESA_framebuffer_flip_x
+
+Contact
+
+    Cici Ruan <ruanc@chromium.org>
+
+Contributors
+
+    Cici Ruan, Google
+    Kristian Kristensen‎, Google
+    Fritz Koenig, Google
+    Rob Clark, Google
+    Chad Versace, Google
+
+Status
+
+    Proposal
+
+Version
+
+    Last Modified Date: April 8, 2020
+    Revision: 1
+
+Number
+
+    OpenGL ES Extension 327
+
+Dependencies
+
+    Requires OpenGL ES 3.1 for FramebufferParameteri.
```

Comment:
Also the minimum GL version / extension information should be given.

Reply:
Added. Thanks!